src/main/java/Tokenise.java

```diff
@@ -48,4 +48,14 @@ public Tokenise(String userInput) {
     public Boolean getIsKeyword() {
```

Comment:
maybe the function name could be renamed to something that sounds more Boolean like just isKeyword instead